opendut-carl/Cargo.toml

```diff
@@ -24,10 +25,9 @@ googletest = { workspace = true }
 http = { workspace = true }
 itertools = { workspace = true }
 pem = { workspace = true, features = ["serde"]}
-oauth2 = { workspace = true, default-features = false, features = ["reqwest"] }
-openidconnect = { workspace = true, default-features = false, features = ["reqwest"] }
 opentelemetry = { workspace = true }
 opentelemetry_sdk = { workspace = true }
+reqwest = { workspace = true, default-features = false, features = ["rustls-tls-native-roots"] }
```

Comment:
See above. This is a change from another commit that does not belong into this PR. To be removed.